src/styles/main.scss

```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Serif&family=Playfair+Display&display=swap');
+
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Playfair Display', serif;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+  gap: 2rem;
+  align-items: center;
+  justify-content: center;
+}
+
+header {
+  margin-bottom: 1rem;
+}
+
+footer {
+  position: relative;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: bisque;
+  padding: 1rem 0;
+  margin-top: 1rem;
+}
+
+#logo {
+  width: 50px;
+}
+
+.center-txt {
+  text-align: center;
+}
+
+.item-image {
+  width: auto;
+}
+
+#items {
+  display: grid;
+  margin-top: 4rem;
+  grid-template-columns: 1fr;
+
+  li {
+    display: flex;
+    flex-direction: column;
+
+    a {
+      margin: 0.5rem 0;
+      align-self: center;
+      border: 1px solid black;
+      padding: 2px;
+      box-shadow: 3px 3px;
+    }
+  }
+}
+
+.name-like {
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+}
+
+.popup {
+  display: none;
+  background-color: rgba($color: #fff, $alpha: 0.8);
+  padding: 1rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
```

Comment:
There are problems with the popup. It disables scrolling on the page completely. Even after closing the popup